telescope/selector.py

```diff
@@ -156,13 +163,21 @@ def _parse_input_for_selectors(self, selector_json):
     multi_selector = MultiSelector()
     multi_selector.start_times = self._parse_start_times(
         selector_json['start_times'])
-    multi_selector.client_providers = selector_json['client_providers']
-    multi_selector.sites = selector_json['sites']
     multi_selector.metrics = selector_json['metrics']
     multi_selector.duration = self._parse_duration(selector_json['duration'])
     multi_selector.ip_translation_spec = self._parse_ip_translation(
         selector_json['ip_translation'])
-
+    
+    if selector_json.has_key('client_providers'):
```

Comment:
Style guide likes `if 'client_providers' in selector_json:`
https://google-styleguide.googlecode.com/svn/trunk/pyguide.html?showone=Default_Iterators_and_Operators#Default_Iterators_and_Operators